ui/js/directives/treeherder/log_viewer.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+treeherder.directive('thLogViewer', ['$sce', '$location', ($sce, $location) => {
+    return {
+        restrict: "E",
+        replace: true,
+        link: function (scope, elem) {
+            elem.on('load', () => {
+                var q = $location.search();
+
+                if (q.highlightStart !== 'undefined' && q.highlightStart) {
+                    scope.logPostMessage({ lineNumber: q.highlightStart });
+                }
+            });
+
+            var searchPart = () => {
+                var q = $location.search();
+
+                return "&highlightStart=" + q.highlightStart + "&highlightEnd=" + q.highlightEnd + "&lineNumber=" +
```

Comment:
@eliperelman Yes we can, but the literal will have a couple of backslashes since template literals records the newline character.